### ISSUE TITLE
Make Our Committees page in main app

### ISSUE THREAD

**Issue by micahdbak** [CSSS/csss-site-frontend#48]
/src/main/index.js
```js
// ...
import { /* ... */, Committees } from './pages';

// ...

const router = createHashRouter([
  // ...
  { path: '/committees', element: <Committees /> },
  // ...
]);
```

/src/main/pages/Committees.js
```js
import React from 'react';
import { Page } from '../components';

export const Committees = () => {
  return (
    <Page>
      {/* ... */}
    </Page>
  );
};
```

**Comment by gunmack** [comment]
@micahdbak for this page, I assume you just want a list of the committees and their descriptions?

**Comment by micahdbak** [comment]
@gunmack yup! I see your WIP PR and it looks good so far; i'll drop the committee info here for now; I think it might be smarter to just put a list of bullet points instead of photos.

- Events Committee
  - The Events Committee meets bi-weekly to discuss and plan the CSSS's various events such as hackathons, workshops, socials, etc. This committee is chaired by the current Director of Events (DoE), and can be attended either on Discord or in person when rooms are booked. Send the current DoE a message on Discord or reach out to them via email at csss-doe-current@sfu.ca to join the Events Committee!
- Multimedia Committee
  - The Multimedia Committee meets asynchronously to work on multimedia for the CSSS such as Instagram posts, posters to be printed and plastered at SFU's campuses, graphics for our website(s), etc. This committee is chaired by the current Director of Multimedia (DoM), and all discussion currently takes place on Discord. Send the current DoM a message on Discord or reach out to them via email at csss-domm-current@sfu.ca to join the Multimedia Committee!
- W3 Committee
  - The W3 Committee meets asynchronously to work on the various websites and projects for the CSSS such as the current website you're looking at, events pages, games such as PacMacro, etc. This committee is chaired by both the current System Administrator (SysAdmin) and the current Webmaster of the CSSS, and all discussion currently takes place on Discord. Send the current SysAdmin or the current Webmaster a message on Discord or reach out to them via email at csss-sysadmin@sfu.ca or csss-webmaster@sfu.ca to join the W3 Committee!
- Policy Committee
  - The Policy Committee starts spontaneously when our constitution or policies come into question. If you think changes should be made, reach out to the current President or Vice President of the CSSS to assemble the Policy Committee and put changes into motion!

**Comment by micahdbak** [comment]
I don't think I'm missing anything - @isabellekwan / @sadjake thoughts?

**Comment by gunmack** [comment]
![image](https://github.com/user-attachments/assets/ec9b6957-ee75-43a9-b599-ea64db09331f)

Is `Get involved` supposed to link to `Our committees.md` ??

In https://github.com/CSSS/csss-site-frontend/pull/64, I did not do this.

Lmk if they are supposed to link, I can do it real quick.
Otherwise close this issue.

@micahdbak

**Comment by jbriones1** [comment]
Closing this in favour of #141
